### DISCUSSION TITLE
Any clever way to make the app open on the same monitor as the mouse cursor (multiple windows)?

### DISCUSSION THREAD

**Discussion by MikeCraig418** [wailsapp/wails#2156] [Q&A]
I was playing with `go-vgo/robotgo` to detect the mouse coordinates.

Anyone have an idea for a simpler way w/o additional libs?

**Comment by ZeronoFreya** [comment]
I will set it to be hidden when the program starts, using the StartHidden parameter. Then get the mouse position, and then display the window, such as this library: `https://github.com/go-vgo/robotgo`，but I haven't tested

**Reply by MikeCraig418** [reply]
Thanks -- I noticed the app will launch on the active monitor. However, when I show/hide the app via a hotkey, it stays on the active monitor.

I've had some issues translating the mouse coordinates to the window coordinates -- but in full transparency, I haven't spent too much time debugging it...

I'm thinking about writing a "launcher" service to listen for the show/hide hot key to launch the app fresh each time.

Much appreciated for your response :)
